src/Content/Changes.php

```diff
@@ -23,86 +27,112 @@
 class Changes
 {
 	/**
-	 * Access helper for the field, in which changes are stored
+	 * Access helper for the cache, in which changes are stored
 	 */
-	public function field(): Field
+	public function cache(): Cache
 	{
-		return $this
-			->site()
-			->version(VersionId::published())
-			->content()
-			->get('changes');
+		return App::instance()->cache('changes');
+	}
+
+	/**
+	 * Returns the cache key for a given model
+	 */
+	public function cacheKey(ModelWithContent $model): string
+	{
+		return match(true) {
+			$model instanceof File => 'files',
+			$model instanceof Page => 'pages',
+			$model instanceof User => 'users'
+		};
```

Comment:
`Suggestion:`

```suggestion
		return $model::CLASS_ALIAS;
```

or if you want it grammatically correct

```suggestion
		return $model::CLASS_ALIAS . 's';
```